app/controllers/admin/api/accounts_controller.rb

```diff
@@ -110,6 +115,12 @@ def buyer_account
     @buyer_account ||= buyer_accounts.find(params[:id])
   end
 
+  def to_present(accounts)
```

Comment:
I would name it something like `preload_associations` to make the purpose more clear.

Reply:
which associations? I wanted it to be clear that this is making it good for being (re)presented. It is not clear which associations it means. But I'm open for another way of thinking.

Reply:
whatever associations that need to be preloaded :grimacing: 

to me `to_present` is too vague and from the name I don't know what the method does.

Reply:
`preload_associations` is more vague. How about `preload_to_present`?

Reply:
So, is this preloading needed specifically for presentation? In this case, `preload_to_present` might be fine. Better than just `to_present` in any case, IMO.

Reply:
> So, is this preloading needed specifically for presentation?

This was what I was trying to explain :)

Reply:
changed